components/compliance-service/reporting/util/csv.go

```diff
@@ -82,3 +83,10 @@ func ReportToCSV(report *reportingapi.Report) (string, error) {
 	}
 	return content, nil
 }
+
+func maxCharLimit(item string) string {
+	if utf8.RuneCountInString(item) >= 32000 {
+		return fmt.Sprintf("character limit exceeds 32,000. truncating message: %s", item[0:31000])
```

Comment:
Is there a reason this is limiting to 31k? I would've expected this to be 31999 or something.

Reply:
I think at most it could be `32,000 - 52` since 52 is the length of the truncation warning.

Reply:
not really a reason, i just figured if we're truncating it we don't need to send all that back, but i can totally update 32k-52!